Neos.Flow/Classes/Http/Headers.php

```diff
@@ -110,8 +110,10 @@ public function set($name, $values, $replaceExistingHeader = true)
             $date = clone $values;
             $date->setTimezone(new \DateTimeZone('GMT'));
             $values = [$date->format('D, d M Y H:i:s') . ' GMT'];
-        } else {
+        } elseif (is_string($values)) {
             $values = (array) $values;
```

Comment:
I have to say, I prefer the `$values = [$values];` version better, because it is more explicit. I really had to check if `(array) $string;` would actually end up in an array with the single string item, or for some php-ish reasons create an array of characters or something else bizarre.